<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QRCondeZxing</class>
 <widget class="QWidget" name="QRCondeZxing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QRCondeZxing</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>选择图片：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>打开</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; 
 }

 QGroupBox::title {
     color:green;
     subcontrol-origin: margin;
     subcontrol-position: top center; 
     padding: 0 3px;
 }</string>
       </property>
       <property name="title">
        <string>二维码</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,8">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>二维码图片：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QGroupBox {
     border: 1px solid gray;
     border-radius: 5px;
     margin-top: 1ex; 
 }

 QGroupBox::title {
     color:green;
     subcontrol-origin: margin;
     subcontrol-position: top center; 
     padding: 0 3px;
 }</string>
       </property>
       <property name="title">
        <string>二维码信息：</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>351</width>
          <height>51</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
